cmd/lakectl/cmd/abuse.go

```diff
@@ -152,6 +153,53 @@ var abuseRandomWritesCmd = &cobra.Command{
 	},
 }
 
+var abuseCommitCmd = &cobra.Command{
+	Use:    "commit <source ref uri>",
+	Short:  "Commits to the source ref repeatedly",
+	Hidden: false,
+	Args:   cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		u := MustParseRefURI("source ref", args[0])
+		amount := MustInt(cmd.Flags().GetInt("amount"))
+		gapDuration := MustDuration(cmd.Flags().GetDuration("gap"))
+
+		Fmt("Source branch: %s\n", u.String())
+		generator := stress.NewGenerator(1, stress.WithSignalHandlersFor(os.Interrupt, syscall.SIGTERM))
+
+		// generate randomly selected keys as input
+		rand.Seed(time.Now().Unix())
+		generator.Setup(func(add stress.GeneratorAddFn) {
+			for i := 0; i < amount; i++ {
+				add(strconv.Itoa(i + 1))
+			}
+		})
+
+		// generate randomly selected keys as input
+		client := getClient()
+		resp, err := client.GetRepositoryWithResponse(cmd.Context(), u.Repository)
+		DieOnErrorOrUnexpectedStatusCode(resp, err, http.StatusOK)
+
+		// execute the things!
+		generator.Run(func(input chan string, output chan stress.Result) {
+			ctx := cmd.Context()
+			client := getClient()
+			for work := range input {
+				start := time.Now()
+				resp, err := client.CommitWithResponse(ctx, u.Repository, u.Ref, &api.CommitParams{},
+					api.CommitJSONRequestBody(api.CommitCreation{Message: work}))
+				if err == nil && resp.StatusCode() != http.StatusOK {
+					err = helpers.ResponseAsError(resp)
+				}
+				output <- stress.Result{
+					Error: err,
+					Took:  time.Since(start),
+				}
+				time.Sleep(gapDuration)
```

Comment:
Support cancellation
```suggestion
				time.Sleep(gapDuration)
				select{
                                     case <-ctx.Done():
                                         return
                                     case <-time.After(gapDuration):
                                 }
```